backend/entityservice/tasks/comparing.py

```diff
@@ -152,20 +151,8 @@ def compute_filter_similarity(chunk_info, project_id, run_id, threshold, encodin
 
     t4 = time.time()
 
-    partial_sparse_result = []
-    # offset chunk's index
-    offset_0, _ = chunk_info_dp1['range']
-    offset_1, _ = chunk_info_dp2['range']
```

Comment:
Offsetting is dealt with by `anonlink.concurrency.process_chunk` so we don’t need this extra step.

Reply:
Nothing sparks joy like deleting code that doesn't spark joy

Reply:
Yes, I was very happy about this!